webcommon/javascript2.jquery/test/unit/src/org/netbeans/modules/javascript2/jquery/JQueryCodeCompletionSelectorsTest.java

```diff
@@ -190,6 +191,7 @@ public void testAttributeName_05() throws Exception {
     }
 
     public void checkAppliedCompletion(final String file, final String caretLine, final String expectedLine, final String itemToComplete, final boolean includeModifiers) throws Exception {
+        RepositoryUpdater.getDefault().waitUntilFinished(10000);
```

Comment:
`checkCompletion` waits for Updater to finish, this check's variant did not. Caused spurious errors / race conditions locally.